runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -714,16 +725,20 @@ private void resetEndpoints()
 	}
 
 	@VisibleForTesting
-	static String formatLevel(int level)
+	static String formatLevel(long level)
```

Comment:
this is no longer a level, its either a level or xp. Maybe something like `format(long number)` is better.

Reply:
I went with formatNumber(long number).